app/controllers/placements/providers/partner_schools_controller.rb

```diff
@@ -14,7 +14,7 @@ def destroy
   private
 
   def set_organisation
-    @provider = current_user.providers.find(params.fetch(:provider_id))
+    @provider = set_provider
   end
```

Comment:
It seems a shame this method is called `set_organisation` but it actually sets `@provider`.

Given the method name, I would have expected it to set `@organisation`, as happens in these other places:

https://github.com/DFE-Digital/itt-mentor-services/blob/89874b12bb0a01ed3526cee2da96fc57a7ac71c1/app/controllers/placements/providers/partner_schools/add_partner_school_controller.rb#L4-L6

https://github.com/DFE-Digital/itt-mentor-services/blob/89874b12bb0a01ed3526cee2da96fc57a7ac71c1/app/controllers/placements/providers/users/add_user_controller.rb#L4-L6

https://github.com/DFE-Digital/itt-mentor-services/blob/89874b12bb0a01ed3526cee2da96fc57a7ac71c1/app/controllers/placements/providers/users_controller.rb#L4-L6

https://github.com/DFE-Digital/itt-mentor-services/blob/89874b12bb0a01ed3526cee2da96fc57a7ac71c1/app/controllers/placements/schools/partner_providers/add_partner_provider_controller.rb#L4-L6